app/assets/stylesheets/components/_event.sass

```diff
@@ -0,0 +1,24 @@
+$Event-backgroundColor: Theme-color(background, default)
+$Event-borderTop-color: #FF623F  // FIXME Put this color on background color MAP on theme.sass file.
```

Comment:
Perhaps `$Event-color-borderTop` ?

This makes me wonder about `$Event-backgroundColor`. Do you think `$Event-color-background` would make more sense?